tests/disas/winch/x64/v128_ops/not.wat

```diff
@@ -18,7 +18,7 @@
 ;;       movq    %rdi, 8(%rsp)
 ;;       movq    %rsi, (%rsp)
 ;;       movdqu  0x19(%rip), %xmm0
-;;       vpcmpeqd %xmm15, %xmm15, %xmm15
+;;       vpcmpeqb %xmm15, %xmm15, %xmm15
```

Comment:
To confirm, changing opcodes here is intentional?

Reply:
It's a "generate all 1s" situation so I just changed it to the lowest lane version, but I'll change it back.